src/main/py/hl7_listener/main.py

```diff
@@ -118,21 +135,54 @@ async def hl7_receiver():
 
 
 async def nc_connect() -> bool:
-    """ Connect to the NATS jetstream server"""
+    """ Connect to the NATS JetStream server"""
     global _nc
     _nc = NATS()
     try:
-        await _nc.connect(_nats_server_url)
+        await _nc.connect(
+            servers=_nats_server_url,
+            nkeys_seed=_nats_nk_file,
+            tls=get_ssl_context(ssl.Purpose.SERVER_AUTH),
+            allow_reconnect=_nats_allow_reconnect,
+            max_reconnect_attempts=_nats_reconnect_attempts,
+        )
         logger.info(logging_codes.NATS_CONNECTED, _nats_server_url)
         return True
```

Comment:
We're either going to return True or raise an exception. imo it would be cleaner to remove the return type.

Reply:
Good point.  Done.